Changelog.json

```diff
@@ -1,4 +1,15 @@
 [
+  {
+    "version": "1.27.0",
+    "date": "November 2nd, 2024",
+    "summary": "Renaming Sudoku strategies to allow for better translations and align with sudoku.com termonology.",
```

Comment:
nit: misspelled terminology and sudoku.com should probably be capitalized

Reply:
Thanks for catching that! I pushed up the fix